config/locales/en.yml

```diff
@@ -324,8 +324,9 @@ en:
       stream_ex2: "Posts in the stream are ordered according to the time of the original post."
       activity: "My activity"
       activity_ex: "Your “My activity” stream shows any posts which you have either liked or commented on. This is a useful way of catching up on any interactions you’ve been having in diaspora*. Posts in this stream are ordered according to the latest activity (comment or like) on each post."
+      activity_ex2: "When you click <span class=\"click\">My activity</span>, two sub-menu items will appear: <span class=\"click\">Liked</span> and <span class=\"click\">Commented</span>. Clicking one of these will show you a stream containing only the posts that you have liked or commented on."
       mentions: "@Mentions"
-      mentions_ex: "This displays any posts made by other people @mentioning you. We’ll discuss @mentions in %{part4_link}."
+      mentions_ex: "This displays any posts made by other people @mentioning you. We’ll discuss @mentions in %{part5_link}."
```

Comment:
I'm not sure how changing one of these links to another part_link would break things. It hasn't done over the past few years. The text strings associated with these links are 'Part 1', 'Part 2', etc, set in https://github.com/diaspora/diaspora-project-site/blob/master/config/locales/en.yml#L239 and should all be translatable.

I think it was @denschub who set this method originally, so let's ask him for comment.